schema/deploy/shipping/views.sql

```diff
@@ -2989,7 +2989,7 @@ create or replace view shipping.uw_reopening_encounters_v1 as
   , site_id
   , details
   from warehouse.encounter
-  where encounter.details @> '{"_provenance": {"redcap": {"url": "https://redcap.iths.org/", "project_id":23854, "event_name":"encounter_arm_1"}}}'
```

Comment:
For encounters, since those aren't getting copied over, I think we'd want this to include both old and new server projects.